### PR TITLE
[RAPTOR-10015] rearrange requirements files

### PR DESCRIPTION
**Pull request by yakov-g**
# This repository is public. Do not put here any private DataRobot or customer's data: code, datasets, model artifacts, .etc.

## Summary

- Add requirements_test_functional.txt and move dr-usertool there.
- remove unused

### CODE REVIEW

**Comment by yakov-g** [comment]
jenkins run test_integration_java_environment

**Comment by yakov-g** [comment]
jenkins run pipelineController:all-expected-tasks

**Review by zohar-mizrahi — COMMENTED** [review]
You decided to provide both `requirements_test_functional.txt` and `requirements_test.txt`. What is the purpose of it? 
Does this mean that whoever works on development doesn't need the functional tests?

**Review by rvorobii — APPROVED** [review]

**Comment by yakov-g** [comment]
jenkins run pipelineController:all-expected-tasks

File: README.md [hunk 1]
```diff
@@ -116,10 +116,14 @@ Rscript -e 'library(caret); install.packages(unique(modelLookup()[modelLookup()$
 
 #### Setting Up Local Env For Testing
 
-1. create Py 3.7 or 3.8 venv
-1. `pip install -r requirements_dev.txt`
-1. `pip install -e custom_model_runner/`
-1. pytest to your heart's content
+1. Create Py venv, >=3.9 is recommended.
```
Comment (eric-s-s) [inline README.md:119]:
```suggestion
1. Create virtualenv  Python>=3.9
```
is this true?  i think we have conflicts somewhere with higher versions.

**Comment by yakov-g** [comment]
> You decided to provide both `requirements_test_functional.txt` and `requirements_test.txt`. What is the purpose of it? Does this mean that whoever works on development doesn't need the functional tests?

I removed requiments_dev, and updated README.
I can not  think of proper instructions regarding functional tests right now.

**Comment by yakov-g** [comment]
jenkins run test_integration_general test_integration_r_environment

File: README.md (hunk 1 above)
Comment (yakov-g) [reply README.md]: done

**Comment by yakov-g** [comment]
jenkins run test_integration_general

**Pull request merged**